source/common/proxy_protocol/proxy_protocol.cc

```diff
@@ -0,0 +1,19 @@
+#include "source/common/proxy_protocol/proxy_protocol.h"
+
+namespace Envoy {
+namespace Common {
+namespace ProxyProtocol {
+
+Network::ProxyProtocolTLVVector
```

Comment:
I think this function can live in tcp_proxy.cc/h. I don't think we need a new directory under common just for this one function.